locationConfig.json

```diff
@@ -16,7 +16,7 @@
         "details": {
             "connector": {
                 "sproxyd": {
-                    "bootstrap": ["localhost:8182"]
+                    "bootstrap": ["localhost:8182:/proxy/arc/"]
```

Comment:
It's actually parsed in sproxydclient, here https://github.com/scality/sproxydclient/blob/master/lib/sproxyd.js#L45.
As to the other point, @LaurenSpiegel, do you have any thoughts?